### PR TITLE
Workflow dispatch

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      
+    - name: Install kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+      
+    - name: Authenticate with Kubernetes cluster
+      run: |
+        mkdir -p "${HOME}/.kube"
+        echo ${{ secrets.KUBE_CONFIG_DATA }} | base64 --decode >  ${HOME}/.kube/config
```
Comment (boddumanohar) [inline .github/workflows/test.yaml:40]: kubeconfig data also changes for a new cluster. You might also have to take kubeconfig from the workflow dispatch.

Comment (geoffrey1330) [reply .github/workflows/test.yaml:40]: I think leaving it as a secret is much better as it needs to be treated as one.
It is easily updated by changing the secret value with the new `kubeconfig`

Comment (geoffrey1330) [reply .github/workflows/test.yaml:40]: No we don't.
we only update it if the cluster we are testing on changes.

**Review by boddumanohar — APPROVED** [review]
Thanks we can go with this changes. Once this is merged, try to run a sample run against the k8s cluster.

**Pull request merged**